packages/py-api/py_api/controllers/hackathon/participants_controller.py

```diff
@@ -27,25 +25,17 @@ def get_all_participants(cls) -> JSONResponse:
                 status_code=404,
             )
 
-        # the dumps funtion from bson.json_util returns a string
+        # the dumps function from bson.json_util returns a string
         return JSONResponse(
             content={"participants": json.loads(dumps(participants))},
```

Comment:
I still believe that this approach to making mongo responses serializiable is a bad one. Unmarshaling and serialializing especially to json is a hefty process. I believe all of this could be achieved with a simple comprehension which remaps the `_id` entry as follows: `id: str(x._id)`

Reply:
Can I open another issue for this, as this PR is massive?